### PR TITLE
web: renderNoResults

### PR DESCRIPTION
**Pull request by jyash97**
## Proposal: 
When there are no `listItems` to be displayed, we can render a message using `onNoResults` prop.

noresults

Thoughts? @lakhansamani @bietkul @metagrover

### CODE REVIEW

**Comment by metagrover** [comment]
We should use `renderNoResults` for this.

**Comment by jyash97** [comment]
Sure will rename this, should I reflect this on other list components as well ?

**Comment by metagrover** [comment]
Yes, go ahead! This will only be available for `next` atm. We can add support for `v2` later if needed.

**Comment by metagrover** [comment]
@jyash97 any update on this?

**Comment by jyash97** [comment]
Will update this in sometime.

**Comment by jyash97** [comment]
Will update by evening

File: packages/web/src/components/list/SingleDataList.js [hunk 1]
```diff
@@ -334,14 +335,15 @@ class SingleDataList extends Component {
 														) || null
 													}
 												>
-													&nbsp;({item.count})
+														&nbsp;({item.count})
```
Comment (metagrover) [inline packages/web/src/components/list/SingleDataList.js]: Indentation

**Comment by metagrover** [comment]
@jyash97 can you add docs for this as well please?

**Comment by jyash97** [comment]
yes, creating a PR for this.

**Pull request merged**